docs/theia.md

```diff
@@ -0,0 +1,68 @@
+# Theia
+
+Theia is the command-line tool which provides access to Theia network flow
+visibility capabilities.
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+- [Compile](#compile)
+- [Usage](#usage)
+<!-- /toc -->
+
+## Installation
+
+Starting with Theia release v0.1, we publish the theia binaries for
```

Comment:
This probably just a personal prefenrence, but to me seems documentation is best written in 3rd person.

E.g.:

`theia` binaries are published for different OS/CPU  Architecture combionations. Refer to the...

Reply:
Sounds good to me, changed.